### PR TITLE
Add sample to "select" an address and generate a URL

### PR DESCRIPTION
**Pull request by jbagga**
Addresses #428 

Naive logic to select the address from a table. Looks for exact match.

cc @rynowak

### CODE REVIEW

File: samples/DispatcherSample/AddressTable.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Routing;
+
+namespace DispatcherSample
+{
+    public class RouteValueAddressTable
```
Comment (rynowak) [inline samples/DispatcherSample/AddressTable.cs]: Add this and the UrlGenerator to services

**Comment by rynowak** [comment]
This looks great. Add the UrlGenerator and the RVAT to services as singletons so that they can be retrieved that way

**Comment by jbagga** [comment]
@rynowak Updated

**Comment by jbagga** [comment]
@rynowak Updated

**Pull request merged**